src/main/resources/schema/ans/0.10.10/utils/variant.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "additionalProperties": false,
+  "description": "Variant metadata describing its ID as well as the websites to which it is assigned",
+  "id": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.10/utils/variant.json",
+  "properties": {
+    "_id": {
+      "$ref": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.10/traits/trait_id.json",
+      "description": "variant ID"
+    },
+    "additional_properties": {
+      "$ref": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.10/traits/trait_additional_properties.json"
+    },
+    "content": {
+      "$ref": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.10/content.json",
+      "description": "Variant content"
+    },
+    "websites": {
```

Comment:
What about putting this into a "conditions" field, which contains a "website_condition" object.  That way it could get extended pretty easily to future conditions.

Reply:
I erred on the side of keeping things simple.  We don't really have a strong idea of what future conditions will look like, so the "websites" field will satisfy the current requirements.  We can expand later by using the JSON schema's "[oneOf](https://json-schema.org/draft/2020-12/json-schema-core.html#rfc.section.10.2.1.3)" keyword to later replace the use of the "websites" field.  This gives us simplicity now, but we'll be able to change things later, while keeping backwards compatibility.